spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TypeConverter.scala

```diff
@@ -60,7 +60,8 @@ trait NullableTypeConverter[T <: AnyRef] extends TypeConverter[T] {
 
 /** Chains together several converters converting to the same type.
   * This way you can extend functionality of any converter to support new input types. */
-class ChainedTypeConverter[T](converters: TypeConverter[T]*) extends TypeConverter[T] {
+class ChainedTypeConverter[T](converters: TypeConverter[T]*) extends NullableTypeConverter[T] {
+  def testAnyVal[T](x: T)(implicit evidence: T <:< AnyVal = null) = evidence != null
```

Comment:
I have no idea, I searched for scala documentation for what this actually does but came up empty. @delyand will have the best explanation as to what it does